src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,109 @@
+.cards-container {
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 50px;
+  padding-left: 40px;
+  width: 1024px;
+  margin: 0 auto;
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 73px;
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  .card {
+    margin-right: 48px;
+    margin-bottom: 46px;
```

Comment:
follow to BEM rules block shouldn't have margins.
move margins to `.cards-container__card` element